plugin/handlergen.go

```diff
@@ -706,9 +706,11 @@ func (p *OrmPlugin) generateStrictUpdateHandler(message *generator.Descriptor) {
 	p.P(`return nil, err`)
 	p.P(`}`)
 
-	p.P(`if count == 0 {`)
-	p.P(`err = `, p.Import(gatewayImport), `.SetCreated(ctx, "")`)
-	p.P(`}`)
+	if p.gateway {
+		p.P(`if count == 0 {`)
```

Comment:
I believe this is the only use of `count`, so the part where it's declared and set above can also be wrapped in a similar `if`. I don't consider that critical though.